x-pack/plugins/apm/public/components/shared/charts/Histogram/__test__/Histogram.test.js

```diff
@@ -87,7 +87,7 @@ describe('Histogram', () => {
       const tooltips = wrapper.find('Tooltip');
 
       expect(tooltips.length).toBe(1);
-      expect(tooltips.prop('header')).toBe('811 - 869 ms');
+      expect(tooltips.prop('header')).toBe('811 - 927 ms');
```

Comment:
Would be nice if these test intervals were easier to read, and we had caught it that way. 
But not necessary for this PR.

Reply:
Can you clarify with an example? Not sure I understand. Maybe it's an easy fix?